feature-libs/checkout/b2b/components/checkout-payment-type/checkout-payment-type.component.html

```diff
@@ -3,79 +3,83 @@ <h2 class="cx-checkout-title d-none d-lg-block d-xl-block">
 </h2>
 
 <ng-container *ngIf="paymentTypes$ | async as paymentTypes">
-  <ng-container
-    *ngIf="
-      !!paymentTypes.length &&
-        (typeSelected$ | async) &&
-        !(isUpdating$ | async);
-      else loading
-    "
-  >
-    <div role="status" [attr.aria-label]="'common.loaded' | cxTranslate"></div>
-    <div class="row">
-      <div class="col-md-12 col-lg-6">
-        <label>
-          <span class="label-content">{{
-            'checkoutB2B.poNumber' | cxTranslate
-          }}</span>
-          <input
-            #poNumber
-            class="form-control"
-            formControlName="poNumber"
-            type="text"
-            placeholder="{{ 'checkoutB2B.placeholder' | cxTranslate }}"
-            value="{{ cartPoNumber$ | async }}"
-          />
-        </label>
-      </div>
-    </div>
-
-    <div class="row">
-      <div class="col-md-12 col-lg-6">
-        <label class="cx-payment-type-container">
-          <span class="label-content">{{
-            'paymentTypes.title' | cxTranslate
-          }}</span>
-          <div class="form-check" *ngFor="let type of paymentTypes">
+  <ng-container *ngIf="!paymentTypesError">
```

Comment:
This condition is the only change made in template: don't display form if an error occurs when fetching PaymentTypes